ckanext/switzerland/helpers/mail_helper.py

```diff
@@ -5,14 +5,25 @@
 from ckan.lib.base import render_jinja2
 import ckan.lib.mailer as mailer
 
+SITE_TITLE = config.get('ckan.site_title', 'opendata.swiss')
+SITE_URL = config.get('ckan.site_url', 'https://opendata.swiss')
+SHOWCASE_ADMIN_EMAIL = config.get(
+    'ckanext.switzerland.showcase_admin_email',
+    'opendata@bfs.admin.ch'
```

Comment:
In general, I would prefer not to have these default values in the code, and we should definitely not include this email address in a public repository.

The site title and url are definitely configured both locally and in our deployed environments, so adding them here as defaults is unnecessary. The showcase admin name is a bit less clear cut, but I would also prefer not to have it hardcoded here. In case the mandant of the project changes again in future, for example, it would have to be changed here as well as in the configuration.